FinalExam/FinalExamTaskSelenium/FinalExamTaskSelenium/UnitTest1.cs

```diff
@@ -0,0 +1,60 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+namespace ExamITAcadeny;
+
+public class Tests
+{
+    private IWebDriver driver;
+    private const string urlEpam = "https://www.epam.com/careers";
+    private const string urlMain = "https://training.by/";
+    private WebDriverWait waiter;
+
+    [SetUp]
+    public void Setup()
+    {
+        driver = new ChromeDriver();
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(urlMain);
+        driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(40);
+        driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(0.1);
+
+        waiter = new WebDriverWait(driver, TimeSpan.FromSeconds(40));
+
+    }
+
+    [Test]
+    public void CheckButtonIsDisplayed()
+    {
+        driver.Navigate().GoToUrl(urlEpam);
+        var butIsdisplayed = driver.FindElement(By.CssSelector("a[href='https://www.epam.com/support-ukraine']")).Displayed;
+        var butIsEnabled = driver.FindElement(By.CssSelector("a[href='https://www.epam.com/support-ukraine']")).Enabled;
+        var butActive = false;
+        if (butIsdisplayed && butIsEnabled)
+        {
+            butActive = true;
+        }
+        Assert.IsTrue(butActive);
```

Comment:
Assert.That(butIsdisplayed && butIsEnabled, Is.True)
And do not forget to use message in assertions :)